paymentsheet/src/main/java/com/stripe/android/paymentsheet/navigation/PaymentSheetScreen.kt

```diff
@@ -75,6 +75,8 @@ internal sealed interface PaymentSheetScreen {
     val walletsDividerSpacing: Dp
     val animationStyle: AnimationStyle
         get() = AnimationStyle.FullPage
+    val showsMandates: Boolean
+        get() = true
```

Comment:
I wonder if this value _shouldn't_ have a default? I think with this current set up it'd be easy to accidentally add a screen which displays mandates but shouldn't